blueprints/automation/panhans/advanced_heating_control.yaml

```diff
@@ -316,6 +316,8 @@ blueprint:
               filter:
                 - domain:
                     - input_boolean
+                    - binary_sensor
+                    - timer
```

Comment:
Timers won't work here. Since they are not binary. To make them work there are some changes in the variable and trigger section needed.

Reply:
Thanks. My use case was binary_sensor and I only tested that one. Removed timer